R/read_table.R

```diff
@@ -44,6 +44,18 @@ read_wikinodes <- function(url, replace_linebreak = ", ", ...) {
 
 read_wikitables <- function(url, ...) {
   read_wikinodes(url) %>%
-    rvest::html_table(fill = TRUE)
+  tryCatch(
+    expr = {
+      rvest::html_table(fill = TRUE)
+    },
+    error = function(e){
+      message('Caught an error!')
```

Comment:
Can we give a more informative error message? *Why* didn't it work? In the Marvel example the problem is that there is a cell that spans multiple rows, so we might say something about that.

Reply:
would this be the only case that returns an error here, or do I need to do an if statement? I was trying to do an if to see if statement to see if the error messaged contained a a string from what this returns, but I'm having issues with the error being an invalid regular expression and not working with grepl